shared/templates/template_BASH_zipl_bls_entries_option

```diff
@@ -1,11 +1,11 @@
 # platform = Red Hat Enterprise Linux 8
 
 # Correct BLS option using grubby, which is a thin wrapper around BLS operations
-grubby --update-kernel=ALL --args="{{{ ARG_NAME_VALUE }}}"
+grubby --update-kernel=ALL --args="{{{ ARG_NAME }}}={{{ ARG_VALUE }}}"
 
 # Ensure new kernels and boot entries retain the boot option
 if [ ! -f /etc/kernel/cmdline ]; then
     echo "{{{ ARG_NAME_VALUE }}}" >> /etc/kernel/cmdline
```

Comment:
One occurrence of `ARG_NAME_VALUE` was left over.

Reply:
Fixed, thanks